config/public-api-v1.yml

```diff
@@ -31,31 +31,441 @@ paths:
+          example: true
+        other_requirements:
+          type: string
+          format: markdown
+          description: |
+            Any non-academic qualifications or documents the applicant may need.
+          example: other_requirements
+        personal_qualities:
+          type: string
+          description: |
+            Any skills, motivation and experience the provider is looking in applicants.
+          example: personal_qualities
+        program_type:
+          type: string
+          description: program_type
+          example: scitt_programme
+          enum:
+            - higher_education_programme
+            - scitt_programme
+            - school_direct_training_programme
+            - school_direct_salaried_training_programme
+            - pg_teaching_apprenticeship
+        provider_code:
+          type: string
+          description: "Unique code for the provider of this course."
+          maxLength: 3
+          example: 6CL
+        qualifications:
+          type: string
+          description: |
+            The qualifications as an outcome of the course.
+          example: [ qts ]
+          enum:
+            - [ qts ]
+            - [ pgce ]
+            - [ pgde ]
+            - [ qts, pgce ]
+            - [ qts, pgde ]
+        recruitment_cycle_year:
+          type: string
+          description: The recruitment cycle that this course is available in.
+          example: 2020
+        required_qualifications:
+          type: string
+          format: markdown
+          description: |
+            The minimum academic qualifications needed for this course.
+          example: required_qualifications
+        required_qualifications_english:
+          type: string
+          description: "English GCSE requirements for applicants."
+          example: equivalence_test
+          enum:
+            - ""
+            - must_have_qualification_at_application_time
+            - expect_to_achieve_before_training_begins
+            - equivalence_test
+            - not_required
+        required_qualifications_maths:
+          type: string
+          description: "Maths GCSE requirements for applicants."
+          example: equivalence_test
+          enum:
+            - ""
+            - must_have_qualification_at_application_time
+            - expect_to_achieve_before_training_begins
+            - equivalence_test
+            - not_required
+        required_qualifications_science:
+          type: string
+          description: "Science GCSE requirements for applicants."
+          example: equivalence_test
+          enum:
+            - ""
+            - must_have_qualification_at_application_time
+            - expect_to_achieve_before_training_begins
+            - equivalence_test
+            - not_required
+        running:
+          type: boolean
+          description: "Is the course currently running."
+          example: true
+        salary_details:
```

Comment:
Caveat

A fee course do not have salary, unless there is an exception.

Reply:
Hence this option is not required, I believe.